### PR TITLE
Crest dropdown

### PR DESCRIPTION
**Pull request by Hootyoo**

### CODE REVIEW

File: crest.md [hunk 1]
```diff
@@ -53,3 +61,32 @@ In order to provide a seamless user experience, _CREST_ exposes a number of even
...
+- The flags of each language.
+- The name of the language(s) as displayed when hovered.
+- The side the selector should show up.
+
+### Dropdown
+
+For websites where more customisation options are required, the dropdown language selector is available. Setup is slightly more involved:
+
+1. Create and empty `span` or `div` with the class `el-dropdown` (or the one configured on the Dashboard).
+2. Add the translator stub with the `?disableSelector` query set to `true` or edit the selector to include this query.
```
Comment (zmeggyesi) [inline crest.md:82]: I really need to _disable_ the selector to insert the dropdown?!

File: crest.md [hunk 2]
```diff
@@ -53,3 +61,32 @@ In order to provide a seamless user experience, _CREST_ exposes a number of even
...
+3. On the Dashboard, under _Language selector_, select JavaScript publishing and Dropdown. Make sure to save the settings. Alternatively, you can copy the embed code available under _Language selector_ and insert it just like you inserted the stub.
```
Comment (zmeggyesi) [inline crest.md:83]: I don't quite follow the alternative path you describe here...

File: crest.md [hunk 3]
```diff
@@ -53,3 +61,32 @@ In order to provide a seamless user experience, _CREST_ exposes a number of even
...
+
+This selector has two main versions. V1 shows the currently selected language and the rest of them are available through a drop-down selector. V2 on the other hand is more compact by default. It shows a globe icon (🌐), then the language choice is available via a more modern dropdown design.
```
Comment (zmeggyesi) [inline crest.md:85]: I'm not sure how I can switch between the two designs, should I need that...

**Review by zmeggyesi — CHANGES_REQUESTED** [review]

File: crest.md (hunk 1 above)
Comment (Hootyoo) [reply crest.md:82]: You *should*. Otherwise, you get two selectors.

**Review by zmeggyesi — APPROVED** [review]

**Pull request merged**